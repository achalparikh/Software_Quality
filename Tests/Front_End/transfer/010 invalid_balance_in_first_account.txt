login
standard 
ACHAL PARIKH	
transfer
00001
00011
2000000000000.00
logout