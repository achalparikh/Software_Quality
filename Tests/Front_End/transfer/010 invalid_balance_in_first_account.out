Enter user type - "standard" or "admin":  
Logging in as standard user - enter name:  
You are now logged in as "ACHAL PARIKH"
COMMAND: transfer - enter FROM account number:
Transferring from 00001 - enter TO account number:
Transferring to 00011 - enter amount:
ERROR: You cannot withdraw that amount as funds are insufficient. Try again
You are now logged out
